datadog/dogstatsd/base.py

```diff
@@ -56,6 +56,21 @@ def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, max_buffer_size=50, nam
         If set, it overrides default value.
         :type DD_DOGSTATSD_PORT: integer
 
+        :envvar DD_ENV: the env of the service running the dogstatsd client.
+        If set, it is appended to the constant tags of the statsd client.
+        It will not override any existing `env` tag.
```

Comment:
I think the phrasing is misleading. It will not override it but will result in duplicate value for the tag. If `env` is passed in `constant_tags` or when calling `gauge` or other metrics we will have this tags multiple times.

The behavior is correct, I just think we should reword the docstrings.

Reply:
:+1: Discussed offline and changed this just now - let me know if it looks better.